### DISCUSSION TITLE
How can I summarize the prometheus rule files to the thanos rule?

### DISCUSSION THREAD

**Discussion by Hello-Linux** [artifacthub/hub#2643] [Q&A] [answered]
**Describe the bug**
I see this values's config file,but don't know How can I summarize the prometheus rule files to the thanos ruler server? the native thanos config is add "--rule-file "/path/to/rules/*.rules.yaml",  and I set   existingConfigmap: "prometheus-kube-prometheus-stack-prometheus-rulefiles-0" but my thanos ruler service show zero rules
![005](https://user-images.githubusercontent.com/17851645/209911927-2540a2d4-cb39-42b6-b0f5-778648d690ee.png)

![002](https://user-images.githubusercontent.com/17851645/209787890-715635ad-7df7-4db0-ae1c-d655d8d8df21.png)

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: centos7

**Smartphone (please complete the following information):**

**Additional context**

**Comment by Hello-Linux** [comment]
@kensipe @idvoretskyi @mattfarina @neilprosser

**Comment by tegioz** [comment, marked as answer]
Hi @Hello-Linux 👋

For issues with the content listed on artifacthub.io it's best to contact the particular project directly. In the links section of the right panel in the package view you may find some links that can point you to the project's repository 🙂
